capstone/src/main/resources/static/user.js

```diff
@@ -344,89 +305,87 @@ function closeForm() {
 
 /* function to get all events hosted by the current organization */
 function getOrganizationEvents() {
-  var userType = sessionStorage.getItem("user-type");
-  if (userType == null) {
-    getUserType(true);
-  }
-  if (userType == "organization") {
-    fetch('get-' + userType).then(response => response.json()).then((data) => {
-      const eventDiv = document.getElementById("created-events");
-      data[0].events.forEach((event) => eventDiv.appendChild(createSavedEventElement(event, false, true)));
-      displayMain(true);
-    });
-  } else {
-    displayMain(false);
-  }
+  fetch('user-info').then(response => response.json()).then((data) => {
+    var userType = data.userType;
+    if (userType == "organization") {
```

Comment:
Same as for "individual", I would avoid the roundtrip to the server just to ensure that the function is being invoked for an organization.